src/firefly/js/visualize/ui/FileUploadViewPanel.jsx

```diff
@@ -112,14 +113,14 @@ export function FileUploadViewPanel({setSubmitText, acceptList, acceptOneItem})
     useEffect(() => {
         dispatchTableAddLocal(summaryModel, undefined, false);
         return (() => {
-            if (!keepState) dispatchTableRemove(summaryTblId);
+            dispatchTableRemove(summaryTblId);
         });
     }, [summaryModel]);
 
     useEffect(() => {
         dispatchTableAddLocal(detailsModel, undefined, false);
         return (() => {
-            if (!keepState) dispatchTableRemove(detailsTblId);
+            dispatchTableRemove(detailsTblId);
```

Comment:
Would this mean `keepState` is no longer supported?

Reply:
I wasn't able to find any examples of `keepState` usages like that in existing code, it introduces problems, so it looks like a misuse of that FieldGroup's feature here